src/app/messages/pl/messages.json

```diff
@@ -210,5 +210,9 @@
   "i18n:toolbar:studyguides:button:text": "Kompendia",
   "i18n:toolbar:studyguides:popup:button:back-to-top": "Idź na górę",
   "i18n:toolbar:studyguides:popup:close-button:aria-label": "Zamknij Kompendia",
-  "i18n:toolbar:studyguides:popup:heading": "Kompendium"
+  "i18n:toolbar:studyguides:popup:heading": "Kompendium",
+  "i18n:toolbar:textresizer:button:aria-label": "Zmień rozmiar tekstu",
+  "i18n:toolbar:textresizer:button:decrease:aria-label": "Zmniejsz rozmiar tekstu",
+  "i18n:toolbar:textresizer:button:increase:aria-label": "Zwiększ rozmiar tekstu",
+  "i18n:toolbar:textresizer:popup:heading": "Rozmiar tekstu"
```

Comment:
@firadecki 👋 Would you have some time to look at these and let me know if they work? I took them from google translate so maybe they need some changes.

Reply:
@jivey Translation looks fine, grammarly correct. There is more than one translation possible for some positions but it's hard to specify without seeing text resizer. Translation is correct and good for now, I'll check if anything can be perfected during testing.